spec/abstractops.html

```diff
@@ -196,31 +196,48 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
     <h1>
       ToTemporalRoundingIncrement (
         _normalizedOptions_: an Object,
-        _dividend_: an integer or *undefined*,
+      ): either a normal completion containing a Number, or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a finite Number of at least 1, and returns that value.
+        It performs no further validation.
+      </dd>
+    </dl>
+    <emu-alg>
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, *"number"*, *undefined*, *1*<sub>𝔽</sub>).
+      1. If _increment_ is *NaN* or *+&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. If _increment_ &lt; *1*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. Return _increment_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-validatetemporalroundingincrement" type="abstract operation">
+    <h1>
+      ValidateTemporalRoundingIncrement (
+        _increment_: a Number,
+        _dividend_: an integer &geq; 1,
         _inclusive_: a Boolean,
-      )
+      ): either a normal completion containing an integer, or an abrupt completion
     </h1>
     <dl class="header">
       <dt>description</dt>
       <dd>
-        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a valid value for the option, and returns that value rounded to an integer.
-        If _dividend_ is *undefined*, any non-zero positive value is valid.
-        Otherwise, the value must not be more than _dividend_ (or must be less than _dividend_, if _inclusive_ is *false*) and must divide evenly into _dividend_.
+        It performs further validation of a value returned from ToTemporalRoundingIncrement.
+        The value _increment_ must not be more than _dividend_ (or must be less than _dividend_, if _inclusive_ is *false*) and must divide evenly into _dividend_.
```

Comment:
I find this description and the corresponding algorithm confusing... what is its purpose?

Reply:
So, I wanted to replace the previous ToTemporalRoundingIncrement operation which IMO does too much (get the value of the property, make sure it's positive, and also make sure that it divides evenly into _dividend_, unless _dividend_ is undefined in which case the operation was broken) and separate it into two parts:
- ToTemporalRoundingIncrement, which gets the property value and does basic conversion and checks (convert to number, make sure the value is positive);
- ValidateTemporalRoundingIncrement, which validates that the value is within the appropriate range, and divides evenly into _dividend_.

The latter is only intended to be called if that validation needs to be done.

Maybe there's a name that conveys that better? e.g., RejectNonDivisorRoundingIncrement? 
I'll make an attempt at rewriting the description more clearly.

The separation will become more important with #2254 when we ensure property accesses are alphabetical; _dividend_ usually depends on the value of the `smallestUnit` property, which comes later in the alphabet than `roundingIncrement`.